app/src/androidTest/java/org/mozilla/focus/activity/URLCompletionTest.java

```diff
@@ -67,4 +91,206 @@ public void CompletionTest() throws InterruptedException, UiObjectNotFoundExcept
         TestHelper.hint.waitForExists(waitingTime);
         assertTrue (TestHelper.inlineAutocompleteEditText.getText().equals("http://www.mozilla.org"));
     }
+
+    @Test
+    public void CustomCompletionTest() throws InterruptedException, UiObjectNotFoundException {
```

Comment:
\[PMD\] INFO: The method 'CustomCompletionTest\(\)' has a NCSS line count of 48\.

Reply:
@boek I actually don't know how to fix this violation - could you help?